src/Foliage-Core/FOBlogPostRenderContext.class.st

```diff
@@ -4,12 +4,36 @@ Class {
 	#category : #'Foliage-Core-Blog'
 }
 
+{ #category : #accessing }
+FOBlogPostRenderContext >> abstract [
+
+	^ self page abstract
+]
+
 { #category : #'as yet unclassified' }
 FOBlogPostRenderContext >> nextPost [ 
 	^ page parent postAfter: self page
 ]
 
+{ #category : #accessing }
+FOBlogPostRenderContext >> pathString [
+
+	^ page pathString
+]
+
 { #category : #'as yet unclassified' }
 FOBlogPostRenderContext >> previousPost [ 
 	^ page parent postBefore: self page
 ]
+
+{ #category : #accessing }
+FOBlogPostRenderContext >> publicationDate [
```

Comment:
This should be available for all pages/published items. This is the switch to make it public